apis/api-journeys/src/scripts/updateSubmitButtons.ts

```
@@ -0,0 +1,94 @@
+import { PrismaClient } from '.prisma/api-journeys-client'
```

ESLint

[error] Useless path segments for ".prisma/api-journeys-client", should be "./.prisma/api-journeys-client"

(import/no-useless-path-segments)